data/movie/Batman Mask of the Phantasm (1993)/Batman The Complete Animated Series Boxset/disc01-summary.txt

```diff
@@ -0,0 +1,40 @@
+Name: Batman: Mask of the Phantasm (1993)
```

Comment:
Something isn't right here. You have this labeled as The Complete Animated Series (which has 12 discs) but you have just one disc with the contents from The Mask of the Phantasm Blu-ray. Did you mean to include the Animated series and The Mask of the Phantasm? If so I would prefer different titles in different pull requests